test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -181,12 +184,23 @@ public async Task StartFakeOcelotWithWebSocketsWithConsul()
 
     public void GivenThereIsAConfiguration(FileConfiguration fileConfiguration)
         => GivenThereIsAConfiguration(fileConfiguration, _ocelotConfigFileName);
-
+
+    public async Task GivenThereIsAConfigurationAsync(FileConfiguration fileConfiguration)
+        => await GivenThereIsAConfigurationAsync(fileConfiguration, _ocelotConfigFileName);
+
+    public async Task GivenThereIsAConfigurationAsync(FileConfiguration from, string toFile)
+    {
+        toFile ??= _ocelotConfigFileName;
+        var jsonConfiguration = JsonConvert.SerializeObject(from, Formatting.Indented);
+        await File.WriteAllTextAsync(toFile, jsonConfiguration);
+        Files.Add(toFile); // register for disposing
+    }
```

Comment:
Simply return the `Task` object as it can be refactored in this manner 👇 
```suggestion
    public Task GivenThereIsAConfigurationAsync(FileConfiguration from, string toFile)
    {
        toFile ??= _ocelotConfigFileName;
        Files.Add(toFile); // register for disposing
        var jsonConfiguration = JsonConvert.SerializeObject(from, Formatting.Indented);
        return File.WriteAllTextAsync(toFile, jsonConfiguration);
    }
```

Reply:
Done!